src/components/orderstatuspopup.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import { Button } from "primereact/button";
+import { Column } from "primereact/column";
+import CustomDialog from "./common/customdialog";
+import CustomTable from "./common/customtable";
+import { ORDER_STATUS_LIST } from "../constants/appConstants";
+import CustomButton from "./Button";
+
+interface OrderStatus {
+  code: number;
+  description: string;
+}
+
+const OrderStatusPopup = () => {
+  const HEADERS = [
+    { field: "code", header: "Status Code" },
+    { field: "description", header: "Description" },
+  ];
+
+  const [orderStatusList, setOrderStatusList] = useState<Array<OrderStatus>>(
+    []
+  );
+  const [openDialog, setOpenDialog] = useState<boolean>(false);
+
+  useEffect(() => {
```

Comment:
useEffect(() => {
    setOrderStatusList(ORDER_STATUS_LIST);
  }, [ORDER_STATUS_LIST]);

Reply:
Is this required? ORDER_STATUS_LIST  is static

Reply:
It's a good practise to include the dependencies.